hydra-node/src/Hydra/Persistence.hs

```diff
@@ -90,7 +91,9 @@ createPersistenceIncremental fp = do
               -- NOTE: We require the whole file to be loadable. It might
               -- happen that the data written by 'append' is only there
               -- partially and then this will fail (which we accept now).
-              case forM (C8.lines bs) Aeson.eitherDecodeStrict' of
-                Left e -> throwIO $ PersistenceException e
-                Right decoded -> pure decoded
+              let loadOrFail i o =
+                    case Aeson.eitherDecodeStrict' o of
+                      Left e -> throwIO $ PersistenceException ("Error at line: " <> show (i + 1) <> " in file " <> fp <> " - " <> e)
+                      Right decoded -> pure decoded
+               in iforM (C8.lines bs) loadOrFail
```

Comment:
@noonio Why use `iforM` instead of `forM` (that is already in our prelude)?

Reply:
I wanted the line number.

Reply:
So we can keep `forM` and not need the import?